mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java

```diff
@@ -206,7 +206,7 @@ public void setAccountBalance(Account account) {
         hideSwipeProgress();
 
         TransitionManager.beginDelayedTransition(homeScreenContainer);
-        mTvAccountBalance.setText(Constants.TAP_TO_REVEAL);
+        mTvAccountBalance.setText(R.string.tap_to_reveal);
```

Comment:
Are you sure it is working correctly? What about if we use`getString(R.string.tap_to_reval)` here and in line numbers `120`, `132` and `165`

Reply:
I checked in output. It went fine. Let me attach the screen shot

Reply:
> Are you sure it is working correctly? What about if we usegetString(R.string.tap_to_reval) here and in line numbers 120, 132 and 165

Anyway, the build has failed so i'll have to look into that too

Reply:
> I checked in output. It went fine. Let me attach the screen shot

Greate, if it works :)

Reply:
> > Are you sure it is working correctly? What about if we usegetString(R.string.tap_to_reval) here and in line numbers 120, 132 and 165
> 
> Anyway, the build has failed so i'll have to look into that too

You have 6 PMD violations

Reply:
How do I solve it?

Reply:
Inside Android Studio in terminal run `./gradlew check` command. When it finished, go to `file:///home/travis/build/openMF/mobile-wallet/mifospay/build/reports/pmd/pmd.html` There you will find what violations you have done.

Reply:
Also, there will be a link to the above file when all checks are done in the terminal. You can find it by clicking on that link. If you need any more help, feel free to message me.

Reply:
Sure, thank you!